sources/packages/web/src/components/common/ManageProgramAndOfferingSummary.vue

```diff
@@ -11,6 +11,7 @@
       <offering-summary
         :programId="programId"
         :locationId="locationId"
+        :canCreateOffering="educationProgram.isActive"
```

Comment:
the variable can be more generic like, `isEditAllowed` and defaulted to false, so that the component can get rid of client type based logic in future.